server.js

```diff
@@ -4,12 +4,29 @@ import cors from 'cors'
 import mongoose from 'mongoose'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
+
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
+ 
+
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140
```

Comment:
Matches the frontent, that's good.  I wonder how you could tell the frontend about these requirements instead of "hardcoding" them in two different places 🤔